### DISCUSSION TITLE
Chart for v2.4.16

### DISCUSSION THREAD

**Discussion by log1cb0mb** [argoproj/argo-helm#1623] [Q&A] [answered]
Hey,

I cannot seem to find exact chart release for argocd v2.4.16

Last chart version for v2.4.x is 5.7.0 and after that its 5.8.0 which is for v2.5.0 onwards. Am I tripping or it just does not exist yet?

**Comment by pdrastil** [comment, marked as answer]
Hi - you are not tripping. The release doesn't exists for that version as 5.7.0 chart used 2.4.15. You can use `global.image.tag` to pull that image if needed.

**Comment by log1cb0mb** [comment]
Righto. When can we expect the release? Not sure if I want to introduce temp change in the deployment if it can wait?

**Reply by pdrastil** [reply]
This release will never happen as Helm is using single publish repository for publishing charts and features that works only with 2.5.x are already in. That's why you can override tag on older version of chart as 5.7.0 or upgrade to newer chart with 2.5.x version.
